flink-runtime/src/test/java/org/apache/flink/runtime/jobgraph/jsonplan/StreamGraphJsonSchema.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.jobgraph.jsonplan;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonCreator;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonIgnore;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Objects;
+
+public class StreamGraphJsonSchema {
+    public static final String FIELD_NAME_NODES = "nodes";
+
+    @JsonProperty(FIELD_NAME_NODES)
+    private List<JsonStreamNodeSchema> nodes;
+
+    @JsonCreator
+    public StreamGraphJsonSchema(@JsonProperty(FIELD_NAME_NODES) List<JsonStreamNodeSchema> nodes) {
+        this.nodes = nodes;
```

Comment:
I am wondering if this class needs an identifier rather than just being a List of nodes. It would make the hash and equals able to check in more detail.

Reply:
This is a helper class in tests. So I think the version is not needed.

Reply:
Good point - thanks